blog: add release 20.10 (#211)
wazo-platform/wazo-platform.org: content/blog/sprint-review-2010.md

Title: Wazo Platform 20.10 Released
Date: 2020-07-20
Author: The Wazo Authors
Category: Wazo Platform
Tags: wazo-platform, development
Slug: release-review-2010
Status: published

Hello Wazo Platform community!

Here is a short review of the Wazo Platform 20.10 release.

## Technical features

* **Asterisk**: Asterisk was upgraded to 17.5.1. See the [Asterisk release announcement](https://www.asterisk.org/downloads/asterisk-news/asterisk-1751-now-available)

> * **Performance**: We have finished removing the use of encryption for all internal HTTP communication between Wazo components. Encryption now only happens when Wazo is sending/receiving data from the outside world.

Reviewer: We should remove this one:
* Already announced in 20.09
* See [my comment about upgrade note](https://github.com/wazo-platform/wazo-platform.org/pull/210/files#r457267942)

Author: Fixed. Thank you!

> * **Multi-tenancy**: We have improved the consistency of the database for multiple tenants. When a tenant is removed, most of the data related to this tenant are now removed.

Reviewer: Oh it's really not true. maybe less than 10% of data related to this tenant is removed. Maybe something like this:
```suggestion
* **Multi-tenancy**: We have fixed some consistency of the database for multiple tenants.
```

But IMO we should remove this one too

Reviewer 2: Or be more specific.
`Users will now get deleted from the database after the tenant gets deleted.`

Author: I added Pascal's suggestion. Thanks!

## Ongoing features

* **SIP lines and trunks**: We are currently rewriting the SIP configuration API to match PJSIP options and be multi-tenant.

> For more details about the aforementioned topics, please see the roadmap linked below.

Reviewer 3: I don't see any link to a roadmap

Author: I reworded to "changelog"

See you at the next sprint review!

## Resources

* [Install Wazo Platform](/install)

* [Upgrade Wazo and Wazo Platform](/uc-doc/upgrade/). Be sure to read the [breaking changes](/uc-doc/upgrade/upgrade_notes#20-10)

Sources:

* [Upgrade notes](/uc-doc/upgrade/upgrade_notes#20-10)

> * [Wazo Platform 20.10 Changelog](https://wazo-dev.atlassian.net/secure/ReleaseNote.jspa?projectId=10011&version=10101)

Reviewer 3: 20.09 change log

Author: Fixed

## Discussion

> Comments or questions in [this forum post](https://wazo-platform.discourse.group/t/blog-wazo-platform-20-10-released).

Reviewer 3: Seems to have a 404
